<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formulario</class>
 <widget class="QDialog" name="formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QStackedWidget" name="pantallas">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>EB Garamond 12 All SC</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>140</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Bienvenido!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>310</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Seleciona qué deseas hacer:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>470</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnPlato">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>EB Garamond 12 All SC</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Añadir platos</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>EB Garamond 12 All SC</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Crear un menú</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnRecetas">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>EB Garamond 12 All SC</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recetas</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnCerrar">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>580</y>
       <width>143</width>
       <height>33</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>EB Garamond 12 All SC</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cerrar la APP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Añadir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cantidades">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="unidades">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_cantidades">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_ingredientes">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="lista_ing">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>121</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ingredientes">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrediente:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAtras">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Atras</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cantidades_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Unidad:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lista_cant">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>121</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ingredientes_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingredientes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ingredientes_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidades:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>input_cantidades</sender>
   <signal>textEdited(QString)</signal>
   <receiver>formulario</receiver>
   <slot>valida()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAtras</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>irAtras()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMenu</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>crearMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_ingredientes</sender>
   <signal>textEdited(QString)</signal>
   <receiver>formulario</receiver>
   <slot>valida()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCerrar</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>cerrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlato</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>anadirPlato()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>add_tolist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRecetas</sender>
   <signal>clicked()</signal>
   <receiver>formulario</receiver>
   <slot>mostrarRecetas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valida()</slot>
  <slot>selvalor()</slot>
  <slot>calcula()</slot>
  <slot>slot1()</slot>
  <slot>add_tolist()</slot>
  <slot>irAtras()</slot>
  <slot>anadirPlato()</slot>
  <slot>mostrarRecetas()</slot>
  <slot>crearMenu()</slot>
  <slot>cerrar()</slot>
 </slots>
</ui>
